packages/esm-patient-forms-app/src/forms/forms-dashboard.component.tsx

```diff
@@ -40,16 +44,27 @@ const FormsDashboard: React.FC<DefaultWorkspaceProps> = () => {
     [currentVisit, htmlFormEntryForms, mutateForms, patientUuid],
   );
 
+  if (Array.isArray(forms)) {
```

Comment:
Use `userHasAccess()` here instead of this, please.

Reply:
@ibacher updated it ,,thanks